docs/src/modules/components/HomeSteps.js

```diff
@@ -199,11 +199,12 @@ function HomeSteps(props) {
             data-ga-event-action="click"
             data-ga-event-label="home-image"
             className={classes.link}
+            aria-label={t('themesButton')}
           >
             <NoSsr>
               <img
                 className={classes.img}
-                alt="themes"
+                alt=""
```

Comment:
Are you sure about this one? As far as I have seen resources online on the topic, they always say to add an alt tag. It seems to create this warning with the Wave extension:

![Capture d’écran 2019-12-09 à 00 05 18](https://user-images.githubusercontent.com/3165635/70398061-bd41d680-1a17-11ea-958a-25464319162d.png)

Reply:
> Are you sure about this one?

Possibly not. I think it was the result of following two separate issues - remove alt text from decorative images, resulting in no text for the link, and that being rectified by aria-label. 

It would be simpler to make the alt-text meaningful, or better, to disable the link for screen readers, since it just duplicates the button below.

Reply:
Thanks for the link! I didn't know that :). I know that ahrefs will report any image that has a missing alt text.

> hide the link for screen readers

This approach sounds better.